installing/installing_sno/install-sno-preparing-to-install-sno.adoc

```diff
@@ -0,0 +1,20 @@
+[id="preparing-to-install-sno"]
+= Preparing to install single node OpenShift
```

Comment:
Remove "OpenShift" from the title.  Perhaps "Preparing to install on a single node".